go.mod

```diff
@@ -3,8 +3,10 @@ module github.com/GoogleContainerTools/kpt
 go 1.16
 
 require (
+	github.com/Azure/go-autorest/autorest/adal v0.9.5 // indirect
 	github.com/cpuguy83/go-md2man/v2 v2.0.0
 	github.com/go-errors/errors v1.4.0
+	github.com/go-swagger/go-swagger v0.27.0 // indirect
```

Comment:
Since we are using `go install` to install the swagger tool, these shouldn't show up in `go.mod`.

Reply:
Removed.